app/service/rpc_service.ts

```diff
@@ -115,6 +117,20 @@ class RpcService {
     ) as Promise<FetchPromiseType<T>>;
   }
 
+  fetchBytestreamFileWithOptions<T extends FetchResponseType = "text">(
+    bytestreamURL: string,
+    invocationId: string,
+    options: BytestreamFileOptions,
```

Comment:
wdyt about tacking this options param onto the existing method params? maybe we can make `init` an optional field of `BytestreamFileOptions` to make it less awkward to pass other options?

Reply:
sorry, dunno why i didn't do that before, guess i didn't have my typescript brain turned on.  i tacked it on, i'll shift the args around separately so that i'm not touching a bunch of random stuff here.